ec2api/api/flow.py

```diff
@@ -0,0 +1,91 @@
+from oslo_config import cfg
+from oslo_log import log as logging
+from ec2api.api import clients
+from ec2api.api import common
+from ec2api.api import ec2utils
+from datetime import datetime
+from ec2api import exception
+from ec2api.i18n import _
+import urllib2
+import pytz
+import json
+import base64
+import ast
+
+field = '{"end_time": "%s" , "select_fields": ["vrouter", "sourcevn", "sourceip", "destvn", "destip", "protocol", "sport", "dport",  "direction_ing", "setup_time", "teardown_time","agg-packets", "agg-bytes", "action", "sg_rule_uuid", "nw_ace_uuid",  "underlay_proto","underlay_source_port","UuidKey"], "start_time": "%s" , "table": "FlowRecordTable",'
+
+Validator = common.Validator
+admin_group = cfg.OptGroup(name='admin_account',
+                                title='admin group ')
+accounts = [ 
+            cfg.StrOpt('account_id',
+                                help=('admin account Id')),
+            cfg.StrOpt('password',
+                                help=('password for admin account Id')),
+            cfg.StrOpt('query_url',
+                                help=('url of analytics query service'))
+]
+
+CONF = cfg.CONF
+CONF.register_group(admin_group)
+CONF.register_opts(accounts, admin_group)
+
+def isInrange(start_time,end_time):
+    s_d = datetime.strptime(start_time, '%d-%m-%Y %H:%M:%S')
+    e_d = datetime.strptime(end_time, '%d-%m-%Y %H:%M:%S')
+    delta= int((e_d - s_d).total_seconds())
+    if delta <= 3600:
+        return True
+    else:
+        return False
+
+def convert_to_now(time):
+    current_time= datetime.now()
+    t_time= datetime.strptime(time, '%d-%m-%Y %H:%M:%S')
+    local = pytz.timezone ("Asia/Kolkata")
+    local_dt = local.localize(t_time, is_dst=None)
+    utc_dt = local_dt.astimezone (pytz.utc)
+    utcdt = utc_dt.replace(tzinfo=None)
+    delta=int( (current_time - utcdt).total_seconds())
+    now_time= 'now-%ss' % (delta)
+    return now_time
+
```

Comment:
we will set log for maximum 15 days so maximum value we will get from user will be 60*60*12*15=1296000.
I will add 15 day condition in start_time and end_time param checking.